test/views/PensionSchemeViewSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.SpecBase
+import forms.PensionSchemeForm
+import org.jsoup.Jsoup
+import play.api.mvc.Call
+import play.api.test.FakeRequest
+import play.api.test.Helpers.running
+import viewmodels.models.PensionSchemeViewModel
+import viewmodels.DisplayMessage
+import views.html.PensionSchemeView
+import org.jsoup.Jsoup
+import play.api.mvc.Call
+
+
+class PensionSchemeViewSpec extends SpecBase {
+
+  val application = applicationBuilder(userAnswers = None).build()
+
+  running(application) {
+    val view = application.injector.instanceOf[PensionSchemeView]
+
+    implicit val request = FakeRequest()
+    implicit val mess = messages(application)
+
+    "PensionsSchemeView" - {
+
+      "view should render correctly" in {
+
+        val form = new PensionSchemeForm()
+        val viewModel = PensionSchemeViewModel(
```

Comment:
You need to test that every value passed into the view model is rendered on the page. So title should be tested, heading should be tested and the onSubmit should also be tested

Reply:
Does the title get passed to the html? I thought it was for the browser tab